src/NetCoreInstaller/Targets/packaging/windows/NetCoreInstaller/variables.wxi

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <?define Manufacturer = Microsoft Corporation?>
+
+    <?define InstallerDataRootFolder=[CommonAppDataFolder]Microsoft .NET Core?>
+    <?define InstallLocationVariable=InstallDir?>
+    <?define InstrumentationKey=7753edb1-c11e-4515-84b1-1df64222323f?>
```

Comment:
Can you switch this to the actual instrumentation key

Reply:
Done